implementations/rust/ockam/ockam_command/src/project/ticket.rs

```diff
@@ -31,6 +32,9 @@ const AFTER_LONG_HELP: &str = include_str!("./static/ticket/after_long_help.txt"
 /// equal to the value of that attribute. If the value is `*` then any name is allowed
 pub const OCKAM_RELAY_ATTRIBUTE: &str = "ockam-relay";
 
+pub const OCKAM_ENROLLER_ATTRIBUTE_KEY: &str = "ockam-role";
+pub const OCKAM_ENROLLER_ATTRIBUTE_VALUE: &str = "enroller";
```

Comment:
Couldn't we reuse the values from 

https://github.com/build-trust/ockam/blob/99a4b8ee78057c856f3e1a2112b854f0510517f5/implementations/rust/ockam/ockam_api/src/authenticator/direct/direct_authenticator.rs#L14

Reply:
great catch! i didn't know these constants already existed in the codebase. will run a find across files next time